themes/githelpers.theme.bash

```diff
@@ -72,7 +72,7 @@ function _git-hide-status {
 
 function _git-status {
   [[ "${SCM_GIT_IGNORE_UNTRACKED}" = "true" ]] && local git_status_flags='-uno'
-  git status --porcelain ${git_status_flags} 2> /dev/null
+  git status --porcelain ${git_status_flags:-} 2> /dev/null
```

Comment:
Same here - I'd rather split the above line into two lines, where `git_status_flags` is always defined with an empty value, and only in the special _ignore untracked_ case, you set it to `-uno`.